ui/src/pages/session-details/playback/capture-details/capture-job/process-now.tsx

```diff
@@ -44,16 +58,20 @@ export const ProcessNow = ({
         label={translate(STRING.PROCESS_NOW)}
         loading={isLoading}
         theme={ButtonTheme.Neutral}
-        onClick={() => {
-          createJob({
-            delay: 0,
-            name: `Capture #${capture.id}`,
-            sourceImage: capture.id,
-            pipeline: pipelineId,
-            projectId: projectId as string,
-            startNow: true,
-          })
-        }}
+        onClick={
+          // @TODO: get the data from the completed processing and print it to console...ngrok
+          getPipelineInfo
+
+          // @TODO: I should create a job which is then processed by ML backend....how does thi createJob work??
```

Comment:
@vanessavmac The createJob function calls the Django / Antenna backend and creates a new processing job for the selected pipeline. Then the Job sends requests to the ML Backend. The Job is a background task in the Antenna backend. 

Here is where the actual request is made. But you shouldn't have to modify this.
https://github.com/RolnickLab/antenna/blob/8e5438e12fe681d4b3edabe23abd4e3dc0439a71/ami/ml/models/pipeline.py#L166